src/main/scala/zio/interop/reactiveStreams/package.scala

```diff
@@ -23,8 +23,8 @@ package object reactiveStreams {
      * produced by the Sink or any error either produced by the Sink or signaled to the subscriber.
      * @param qSize The size used as internal buffer. If possible, set to a power of 2 value for best performance.
      */
-    def toSubscriber(qSize: Int = 16): ZIO[R, Nothing, (Subscriber[A], Task[B])] =
-      Adapters.sinkToSubscriber(sink, qSize)
+    def toSubscriber[R1 <: R](qSize: Int = 16)(subscribe: Subscriber[A] => ZIO[R1, E, Unit]): ZIO[R1, Throwable, B] =
```

Comment:
Scaladoc needs an update here.

Reply:
done

Reply:
So maybe we  can return `ZManaged[(Subscriber[A], ZIO[R1, Throwable, B])]` here?